edk2toolext/tests/test_self_describing_environment.py

```diff
@@ -81,22 +81,6 @@ def tearDownClass(cls):
     def test_identical_environments(self):
         scopes = ("corebuild", "project_mu")
```

Comment:
what happened here?

Reply:
It was cloning basecore twice and it just takes forever. I need to rewrite it to be better

Reply:
Okay- this test wasn't even really testing the SDE because the SDE is cached in a global singleton. If this was working, it would have failed because the two environments don't match because they're in different folders